### PR TITLE
feat: add more logo

### PR DESCRIPTION
**Pull request by greedythib**

### CODE REVIEW

File: ERC20_LIST.json [hunk 1]
```diff
@@ -7,68 +7,76 @@
...
+                "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/G-Uni_agEUR_USDC.svg"
               },
               "0x26C2251801D2cfb5461751c984Dc3eAA358bdf0f": {
                 "name": "Gelato Uniswap agEUR/wETH LP",
                 "decimals": 18,
                 "symbol": "G-Uni agEUR/wETH",
                 "isSanToken": false,
-                "hasPermit": false
+                "hasPermit": false,
+                "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/G-Uni_agEUR/wETH.svg"
```
Comment (GuillaumeNervoXS) [inline ERC20_LIST.json:62]: they don't exist

File: ERC20_LIST.json [hunk 2]
```diff
@@ -7,68 +7,76 @@
...
+                "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/FRAX.svg"
               },
               "0x2bD9F7974Bc0E4Cb19B8813F8Be6034F3E772add": {
                 "name": "Gelato Uniswap agEUR/USDC LP",
                 "decimals": 18,
                 "symbol": "G-Uni agEUR/USDC",
                 "isSanToken": false,
-                "hasPermit": false
+                "hasPermit": false,
+                "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/G-Uni_agEUR_USDC.svg"
```
Comment (GuillaumeNervoXS) [inline ERC20_LIST.json:54]: it doesn't exist

Comment (greedythib) [reply ERC20_LIST.json:54]: @GuillaumeNervoXS what does not exist?

File: ERC20_LIST.json (hunk 1 above)
Comment (GuillaumeNervoXS) [reply ERC20_LIST.json:62]: they are no svg at this path + typo '/wETH' instead of '_wETH'

**Review by GuillaumeNervoXS — APPROVED** [review]

**Pull request merged**